qa/runtest.sh

```diff
@@ -61,35 +54,35 @@ docker run -d \
     --name "${DB_CONTAINER_NAME}" "${POSTGRES_IMAGE_NAME}"
 DB_CONTAINER_IP=$(docker inspect --format "{{.NetworkSettings.Networks.${DOCKER_NETWORK}.IPAddress}}" ${DB_CONTAINER_NAME})
 
+# TODO: this is duplicating code with server's runtest, we should refactor
 echo "Waiting for postgres to fully initialize"
-set +x
 for i in {1..10}; do
-  retcode=$(curl http://${DB_CONTAINER_IP}:5432 &>/dev/null || echo $?)
-  if test "$retcode" == "52"; then
+  set +e
```

Comment:
Do you want to do both +e and -e in a loop?

Reply:
yeah, otherwise entire script will exit if `docker exec -it "${DB_CONTAINER_NAME}" bash -c pg_isready` returns non zero value.

Reply:
Yes, I understand that but can it not be done outside the loop